diff -durN plus4emu-1.2.9.2.orig/SConstruct plus4emu-1.2.9.2/SConstruct
--- plus4emu-1.2.9.2.orig/SConstruct	2008-09-25 00:04:22.000000000 +0200
+++ plus4emu-1.2.9.2/SConstruct	2012-04-09 14:17:07.165952212 +0200
@@ -76,6 +76,7 @@
         if not plus4emuGLGUIEnvironment.ParseConfig(
             '%s --use-gl --use-images --cxxflags --ldflags' % fltkConfig):
             raise Exception()
+        plus4emuGLGUIEnvironment.Append(LIBS = ['GL'])
     except:
         print 'WARNING: could not run fltk-config'
         plus4emuGLGUIEnvironment.Append(LIBS = ['fltk_images', 'fltk_gl'])
