diff -durN plus4emu-1.2.9.2.orig/SConstruct plus4emu-1.2.9.2/SConstruct
--- plus4emu-1.2.9.2.orig/SConstruct	2008-09-25 00:04:22.000000000 +0200
+++ plus4emu-1.2.9.2/SConstruct	2014-01-01 10:25:47.709279732 +0100
@@ -214,9 +214,9 @@
 else:
     haveSDL = 0
 if not disableLua:
-    haveLua = configure.CheckCHeader('lua.h')
-    haveLua = haveLua and configure.CheckCHeader('lauxlib.h')
-    haveLua = haveLua and configure.CheckCHeader('lualib.h')
+    haveLua = configure.CheckCHeader('lua-5.1/lua.h')
+    haveLua = haveLua and configure.CheckCHeader('lua-5.1/lauxlib.h')
+    haveLua = haveLua and configure.CheckCHeader('lua-5.1/lualib.h')
 else:
     haveLua = 0
 haveZLib = configure.CheckCHeader('zlib.h')
@@ -378,7 +378,7 @@
         plus4emuEnvironment.Append(LIBS = ['mingwex'])
     plus4emuEnvironment.Append(LIBS = ['dotconf'])
 if haveLua:
-    plus4emuEnvironment.Append(LIBS = ['lua'])
+    plus4emuEnvironment.Append(LIBS = ['lua-5.1'])
 if haveSDL:
     plus4emuEnvironment.Append(LIBS = ['SDL'])
 plus4emuEnvironment.Append(LIBS = ['portaudio', 'sndfile'])
diff -durN plus4emu-1.2.9.2.orig/src/script.cpp plus4emu-1.2.9.2/src/script.cpp
--- plus4emu-1.2.9.2.orig/src/script.cpp	2007-12-08 19:44:27.000000000 +0100
+++ plus4emu-1.2.9.2/src/script.cpp	2014-01-01 10:30:28.472265344 +0100
@@ -25,9 +25,9 @@
 
 #ifdef HAVE_LUA_H
 extern "C" {
-#  include "lua.h"
-#  include "lauxlib.h"
-#  include "lualib.h"
+#  include "lua-5.1/lua.h"
+#  include "lua-5.1/lauxlib.h"
+#  include "lua-5.1/lualib.h"
 }
 #endif  // HAVE_LUA_H
 
diff -durN plus4emu-1.2.9.2.orig/src/script.hpp plus4emu-1.2.9.2/src/script.hpp
--- plus4emu-1.2.9.2.orig/src/script.hpp	2007-12-08 19:44:27.000000000 +0100
+++ plus4emu-1.2.9.2/src/script.hpp	2014-01-01 10:30:45.098264492 +0100
@@ -26,9 +26,9 @@
 
 #ifdef HAVE_LUA_H
 extern "C" {
-#  include "lua.h"
-#  include "lauxlib.h"
-#  include "lualib.h"
+#  include "lua-5.1/lua.h"
+#  include "lua-5.1/lauxlib.h"
+#  include "lua-5.1/lualib.h"
 }
 #else
 struct  lua_State;      // dummy declaration
