diff -dur plus4emu-1.2.9.orig/SConstruct plus4emu-1.2.9/SConstruct
--- plus4emu-1.2.9.orig/SConstruct	2008-08-27 15:32:45.000000000 +0200
+++ plus4emu-1.2.9/SConstruct	2008-08-28 05:37:35.000000000 +0200
@@ -1,5 +1,6 @@
 # vim: syntax=python
 
+import os
 import sys
 
 win32CrossCompile = 0
@@ -10,17 +11,7 @@
 enableDebug = 0
 buildRelease = 1
 
-compilerFlags = ''
-if buildRelease:
-    if linux32CrossCompile or win32CrossCompile:
-        compilerFlags = ' -march=pentium2 -mtune=generic '
-if enableDebug and not buildRelease:
-    compilerFlags = ' -Wno-long-long -Wshadow -g -O2 ' + compilerFlags
-    compilerFlags = ' -Wall -W -ansi -pedantic ' + compilerFlags
-else:
-    compilerFlags = ' -Wall -O3 ' + compilerFlags
-    compilerFlags = compilerFlags + ' -fno-inline-functions '
-    compilerFlags = compilerFlags + ' -fomit-frame-pointer -ffast-math '
+compilerFlags = os.environ['CXXFLAGS']
 
 fltkConfig = 'fltk-config'
 
